di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -179,7 +179,7 @@ private String getAuthorizeRedirect(QueryParamsMap queryMap, String errorType)
                 new AuthorizationRequest.Builder(
                                 new ResponseType(ResponseType.Value.CODE),
                                 new ClientID(isMfaReset ? AUTH_CLIENT_ID : ORCHESTRATOR_CLIENT_ID))
-                        .state(ORCHESTRATOR_STUB_STATE)
+                        .state(isMfaReset ? AUTH_STUB_STATE : ORCHESTRATOR_STUB_STATE)
```

Comment:
This was a fix to ensure we have the state attribute at the highest level of the object